### PR TITLE
Allow bru.setNextRequest() on Test Scripts (#2155)

### PR DESCRIPTION
**Pull request by BruAlcaraz**
# Description

Allow the use of bru.setNextRequest() at the test script level
#2155 

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

**Comment by BruAlcaraz** [comment]
@helloanoop could you review this fix?

**Comment by BruAlcaraz** [comment]
@Its-treason and @dcoomber Could you guys review this fix?

File: packages/bruno-electron/src/ipc/network/index.js
Comment (Its-treason) [inline packages/bruno-electron/src/ipc/network/index.js:1050]: This also needs to be added to the CLI code here: 

https://github.com/usebruno/bruno/blob/0988f2b86eae44db4af41169ce19c08339ac0161/packages/bruno-cli/src/runner/run-single-request.js#L342

**Review by Its-treason — CHANGES_REQUESTED** [review]

File: packages/bruno-electron/src/ipc/network/index.js
Comment (BruAlcaraz) [reply packages/bruno-electron/src/ipc/network/index.js:1050]: I have added the same to the CLI as requested but I am not sure how I can test that to make sure it is working as expected.

Comment (BruAlcaraz) [reply packages/bruno-electron/src/ipc/network/index.js:1050]: @Its-treason and @martinsefcik can you please take a look again? I have done the suggested changes.

**Review by Its-treason — APPROVED** [review]

**Comment by BruAlcaraz** [comment]
@helloanoop, @Its-treason and  @dcoomber what would be the next step to get these merged and released?

**Comment by Its-treason** [comment]
This is all you can do, you have to wait for the Maintainer to merge this. CC: @sanjai0py

**Comment by sanjai0py** [comment]
Hey @BruAlcaraz, thanks for taking the time to work on this PR, and thanks @Its-treason for the reviews! This will be released in the next outgoing release (in a week).

**Pull request merged**
